.changeset/two-candles-yawn.md

```
@@ -0,0 +1,5 @@
+---
+'@clerk/clerk-js': patch
+---
+
+Reverts the changes done https://github.com/clerk/javascript/pull/7105, as it was causing JWT's returned from client piggybacking not get inserted into cache even though their claims have actually changed
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)